mbuild/formats/lammpsdata.py

```diff
@@ -5,6 +5,7 @@
 
 import numpy as np
 from parmed.parameters import ParameterSet
+from parmed.structure import Structure
```

Comment:
Small change for RTD build fix

```suggestion
from parmed import Structure
```